packages/@vue/cli-plugin-unit-jest/generator/index.js

```diff
@@ -77,6 +77,19 @@ module.exports = (api, _, __, invoking) => {
 
   if (api.hasPlugin('eslint')) {
     applyESLint(api)
+
+    api.extendPackage({
+      eslintConfig: {
+        overrides: [
+          {
+            files: ['**/__tests__/*.js', '**/__tests__/*.jsx', '**/__tests__/*.ts', '**/__tests__/*.tsx'],
```

Comment:
Seems too verbose. What about `'**/__tests__/*.{j,t}s?(x)'`?

Reply:
I initially tried that but it seems that eslint only supports basic wildcards with *

I may be wrong but couldn't get it to work with glob/regexp matching syntax

Reply:
ESLint uses `minimatch` underlyingly. As I tested, the abovementioned glob works well.

Reply:
I see. Will adjust.

Reply:
Done